app/models/manageiq/providers/google/inventory/collector/network_manager.rb

```diff
@@ -11,8 +11,8 @@ def cloud_subnets
       @subnetworks += connection.networks.select { |x| x.ipv4_range.present? }.map do |x|
         Fog::Compute::Google::Subnetwork.new(
           :name               => x.name,
-          :gateway_address    => x.gateway_ipv4,
-          :ip_cidr_range      => x.ipv4_range,
+          :gateway_address    => x.gateway_i_pv4,
```

Comment:
Even though there is an alias for this method https://github.com/fog/fog-google/blob/af528d0/lib/fog/compute/google/models/network.rb#L16 we were still hitting a no method error:
```
[----] E, [2019-10-30T10:50:08.321536 #16371:d765f4] ERROR -- : MIQ(ManageIQ::Providers::Google::NetworkManager::Refresher#refresh) EMS: [Google Network Manager], id: [3] Refresh failed
[----] E, [2019-10-30T10:50:08.321950 #16371:d765f4] ERROR -- : [NoMethodError]: undefined method `gateway_ipv4' for #<Fog::Compute::Google::Network:0x000000000d8f9500> Method:[block (2 levels) in <class:LogProxy>]
[----] E, [2019-10-30T10:50:08.322065 #16371:d765f4] ERROR -- : /usr/local/lib/ruby/gems/2.5.0/bundler/gems/manageiq-providers-google-a68a49912f5c/app/models/manageiq/providers/google/inventory/collector/network_manager.rb:14:in `block in cloud_subnets'
```

Reply:
Reported at https://github.com/fog/fog-google/issues/478, but no response yet.